evaluate/repository.go

```diff
@@ -19,34 +19,42 @@ import (
 	"github.com/symflower/eval-dev-quality/util"
 )
 
-// Repository evaluate a repository with the given model and language.
-func Repository(logger *log.Logger, resultPath string, model evalmodel.Model, language language.Language, testDataPath string, repositoryName string) (repositoryAssessment metrics.Assessments, problems []error, err error) {
-	log, logClose, err := log.WithFile(logger, filepath.Join(resultPath, evalmodel.CleanModelNameForFileSystem(model.ID()), language.ID(), repositoryName+".log"))
+// Repository holds data about a repository.
+type Repository struct {
+	// Name holds the name of the repository.
+	Name string
+	// DataPath holds the path to the repository.
```

Comment:
Absolute or relative? Just asking because we usually also have this `TestdataPath` that makes all the repository paths relative.

Reply:
absolute. changed the docstring for the field